enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -13,7 +13,7 @@
 from requests.exceptions import HTTPError
 from rest_framework import serializers
 
-from enterprise_access.apps.content_assignments.api import get_content_metadata_for_assignments
+from enterprise_access.apps.content_assignments.utils import get_content_metadata_for_assignments
```

Comment:
Kira and I recently moved this function into a `content_assignments/content_metadata_api.py` file, it should be there if you rebase onto `main`.